service/src/main/java/bio/terra/workspace/common/logging/model/ActivityFlight.java

```diff
@@ -33,8 +32,6 @@
  */
 public enum ActivityFlight {
   // Workspace
-  WORKSPACE_CREATE_FLIGHT(
```

Comment:
🤔 Will this break any in progress flights at the time of rollout?

Reply:
As discussed in standup, I have pulled out the flight deletion and will put it in a separate PR after this one gets to production.